src/views/verify.tsx

```diff
@@ -1,9 +1,19 @@
 import { isValid, openAttestationVerifiers, verificationBuilder, VerificationFragment } from "@govtechsg/oa-verify";
+import { providers } from "ethers";
 import React, { useEffect, useState } from "react";
 import { FailedAlert, SucceedAlert } from "../components/alert";
 import { Status } from "../shared";
 
-type Network = "ropsten" | "homestead" | "rinkeby" | "goerli";
+enum SupportedNetwork {
+  ropsten = "Ropsten",
+  homestead = "Mainnet",
+  rinkeby = "Rinkeby",
+  goerli = "Goerli",
+  maticmum = "Mumbai",
+}
```

Comment:
Will the value here be used anywhere or just the key here will be used?

Reply:
both will be used, key for the [network](https://github.com/Open-Attestation/toolkit/pull/23/files#diff-b1168b7c3bb048557e5d144b742daa82f333df068f09d3aeea0e69f549b73df0R36), while value for the button [labels](https://github.com/Open-Attestation/toolkit/pull/23/files#diff-b1168b7c3bb048557e5d144b742daa82f333df068f09d3aeea0e69f549b73df0R39).

no this repo is not decentralised renderer template. it an [UI](https://toolkit.openattestation.com/) to run some of methods provided by OA suite.

Reply:
Ahh... I see I see.👀

Since the value is used here as a label, do you think it will help if we change "Mumbai" to "Polygon Mumbai" just so it's clear that this is a polygon testnet?

Reply:
right.. maybe i separate the testnets and production nets, see if that is clearer

![Screenshot 2022-09-06 at 4 36 05 PM](https://user-images.githubusercontent.com/4774314/188588024-31f328cb-2994-41e2-8da0-f4e9ba503b39.png)

Reply:
Nice!! This looks good, easy to see!👍